### PR TITLE
feat: remove --disable-component-update from default args

### PR DESCRIPTION
**Pull request by andrea11**
Similarly to puppeteer/puppeteer#13010, 
Quoting @mathiasbynens:

> Summarizing the “why” here:  
The --disable-component-update command-line flag does two things: it disables component updates (desired) but it also prevents any bundled components from being initialised (undesired).

Disable this flag grant more flexibility, and it can always be disabled via the `web-ext run --arg option`

### CODE REVIEW

**Comment by andrea11** [comment]
Can someone review this PR? (@willdurand maybe?)

**Review by Rob--W — APPROVED** [review]
I'd prefer to not auto-download a bunch of extra stuff, but since it is currently possible to add it again, but not to opt out, I reluctantly approve this change.

When merged, this will fix https://github.com/mozilla/web-ext/issues/3093

**Comment by willdurand** [comment]
> I'd prefer to not auto-download a bunch of extra stuff, but since it is currently possible to add it again, but not to opt out, I reluctantly approve this change.

I am not clear on what that means, @Rob--W.

**Comment by Rob--W** [comment]
> > I'd prefer to not auto-download a bunch of extra stuff, but since it is currently possible to add it again, but not to opt out, I reluctantly approve this change.
> 
> I am not clear on what that means, @Rob--W.

I am approving this change, even if it means that Chrome will auto-download some additional components after startup. This will utilize extra bandwidth and CPU when launching (new) browser profiles instances. In exchange for that downside, the upside is that the launched Chrome browser will more closely match production. One example of that is that websites requiring DRM will work.

If devs want to revert to the behavior from before, they can manually add the command line flag to `web-ext run`.

TL;DR: LGTM, let's merge.
